www/package.json

```diff
@@ -75,13 +75,13 @@
     "autoprefixer": "^10.4.7",
     "docusaurus-plugin-typedoc": "^1.0.0-next.12",
     "dotenv": "^16.0.1",
-    "eslint": "^8.40.0",
+    "eslint": "^8.56.0",
     "next": "^14.0.1",
     "next-auth": "^4.22.1",
     "npm-run-all": "^4.1.5",
     "oauth": "^0.10.0",
     "postcss": "^8.4.14",
-    "prettier": "^2.8.8",
+    "prettier": "npm:prettier@^2.8.8",
```

Comment:
I pinned this to prettier 2 -- @ixahmedxi see if we can just remove it here instead somehow. Idk how/why prettier is included in docusaurus' build